manifests/2.5.0/opensearch-dashboards-2.5.0.yml

```diff
@@ -12,27 +12,27 @@ components:
     ref: '2.5'
   - name: searchRelevanceDashboards
     repository: https://github.com/opensearch-project/dashboards-search-relevance.git
-    ref: 2.x
+    ref: '2.5'
```

Comment:
Seems like this repo doesn't cut 2.5 branch yet.

Reply:
I already contacted their release owner she is on it now.
@mingshl

Reply:
cut 2.5 branch https://github.com/opensearch-project/dashboards-search-relevance/tree/2.5

Reply:
Thanks @mingshl !